coding/javascript.md

```diff
@@ -70,6 +70,14 @@ To ensure code formatting consistency across our codebase, we use [Prettier](htt
 
 JavaScript should also be written using [AirBnB](https://github.com/airbnb/javascript) style. New projects can adhere to this by selecting 'AirBnB' during `eslint init`, and existing projects can find the relevant ESLint config in [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb).
 
+### Commenting code
+
+As a general rule, human-readable code is preferred over brevity.
+
+Inline comments should be provided where logic or behaviour is unexpected or required due to external factors.
```

Comment:
This is a general principal, not really specific to javascript.

Reply:
Yes it is a general principle, but if it applied to Python I think it would be nice if it were a little more encouraging of comments:

``` markdown
Inline comments should be provided in any case where it is not immediately obvious what the code is doing.
```

Reply:
It is general, but I think it's good to have here too